tailwind.config.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
           '75%': { transform: 'rotate(-1.5deg)' },
         },
       },
+      gridTemplateColumns: {
+        'auto-fill-award-grid': 'repeat(auto-fill, minmax(52px, 1fr))',
+      },
```

Comment:
While functional, this is not utility-first, so it goes against Tailwind's internal design principles. To better illustrate, this would be utility first if `award-grid` were a themeable unit of measurement, but this is more targeted towards a domain concern.

Instead, because it's a domain concern, JIT values are a better fit.

```
class="grid-cols-[repeat(auto-fill,minmax(52px,1fr))]"
```

Reply:
I didn't realise we could do JIT, very cool. Moved the auto-fill out of the config.